pulp_gem/app/tasks/publishing.py

```diff
@@ -122,20 +121,22 @@ def publish(repository_version_pk):
         gemspecs = []
         for content in (
             GemContent.objects.filter(pk__in=publication.repository_version.content)
-            .only("name", "version")
+            .only("name", "version", "platform")
             .order_by("-pulp_created")
             .iterator()
         ):
             if content.prerelease:
```

Comment:
Never noticed before that we break the assumption of this `only` on the very next line by trying to access the `prerelease` field. This probably has negative impact on publishing.

Reply:
indeed.